book/src/volume-expansion.md

```diff
@@ -4,7 +4,8 @@
 
 Status | Min K8s Version | Max K8s Version | external-resizer Version
 --|--|--|--
-Alpha | 1.14 | - | 0.1
+Alpha | 1.14 | 1.15 | 0.2
```

Comment:
Note to reviewers: I'm assuming here that v0.2 is compatible with both 1.14 and 1.15.